<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DatabaseConnectionWgt</class>
 <widget class="QWidget" name="DatabaseConnectionWgt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>321</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>数据库连接</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,3">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>主机：</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="ip_line">
       <property name="text">
        <string>127.0.0.1</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="1,3">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>端口：</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="port_line">
       <property name="text">
        <string>3306</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,3">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>数据库类型：</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="sqlType_combobox">
       <item>
        <property name="text">
         <string>Mysql</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,3">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>用户名：</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="root_line">
       <property name="text">
        <string>root</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="1,3">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>密码：</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="password_line">
       <property name="text">
        <string>123456</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="1,3">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>数据库名称：</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="databaseName_line">
       <property name="text">
        <string>itcast</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="yes_btn">
       <property name="text">
        <string>确定</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancel_btn">
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
